Dockerfile.xlarge

```diff
@@ -33,10 +33,11 @@ WORKDIR /home/docsworker-xlarge
 # install snooty parser
 RUN python3 -m pip uninstall -y snooty
 RUN python3 -m pip install --upgrade pip flit
-RUN git clone https://github.com/mongodb/snooty-parser.git snooty-parser && \
+RUN git clone https://github.com/mongodb/snooty-parser.git && \
 	cd snooty-parser && \
-	git fetch --all && \
-	git reset --hard origin/master && \
+	git fetch --tags && \
+	latestTag=$(git describe --tags `git rev-list --tags --max-count=1`) && \
+	git checkout $latestTag && \
```

Comment:
In theory this should be safe, but it's generally a best practice to wrap shell variables in double-quotes; e.g. `git checkout "$latestTag"`

Reply:
changed, lemme know if ok.